src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/model/in/ExceptionRecord.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.in;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.ocrvalidation.model.in.OcrDataField;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class ExceptionRecord {
+
+    public final String caseTypeId;
+    public final String poBox;
+    public final String jurisdiction;
+    public final JourneyClassification journeyClassification;
+    public final LocalDateTime deliveryDate;
+    public final LocalDateTime openingDate;
+    public final List<ScannedDocument> scannedDocuments;
+    public final List<OcrDataField> ocrDataFields;
+
+    public ExceptionRecord(
+        @JsonProperty("case_type_id") String caseTypeId,
+        @JsonProperty("po_box") String poBox,
+        @JsonProperty("jurisdiction") String jurisdiction,
```

Comment:
in tech spec, we have this field as `po_box_jurisdiction`. are you going to change it in tech spec?

Reply:
I've assumed it was a bug in the spec. But apparently not, just checked with @lgonczar 
Updated code.